src/_locales/fr/messages.json

```
@@ -11,7 +11,7 @@
     "hash": "8c716f7803f220414804500b514a1dc0"
   },
   "extensionDescription": {
-    "message": "Simplifie le suivi ou l´interaction avec d´autres utilisateurs sur des instances (Mastodon) extérieures.",
+    "message": "Simplifie le suivi ou l´interaction d´autres personnes sur des instances (Mastodon) externes.",
```

What would "interaction" refer to? Other people's interactions, or interactions with other people?

---

Some context: In the original sense/wording, it should refer to people interacting. In English it kinda does not make a difference whose interactions those are:

https://github.com/rugk/mastodon-simplified-federation/blob/94c0a55c9771a53f3639d9e632a41396f699eca6/src/_locales/en/messages.json#L12

So the add-on simplifies interacting with users on other (remote) instances.

---

Thanks for the context
I think we should keep "avec" (with) in that case; "l´interaction d´autres personnes" sounds to me like other people interacting than (the user) interacting with people.

---

```suggestion
    "message": "Simplifie le suivi ou l´interaction d´autres avec d´autres utilisateurs sur des instances (Mastodon) externes.",
```

So like this? NOte you can do inline-suggestions, so the OP @le-jun can apply them with one click. :slightly_smiling_face:

---

Ok, modified it. Kept the "avec", don’t really know why I deleted it in the first place, while keeping the "personnes" which is genderless but closer to "people" than "users".

---

> So like this? Note you can do inline-suggestions, so the OP le-jun can apply them with one click. 🙂

Oh, I forgot about that one
Thanks for the reminder 😊




> Ok, modified it. Kept the "avec", don’t really know why I deleted it in the first place, while keeping the "personnes" which is genderless but closer to "people" than "users".

I think "personnes" sounds better and is more inclusive, but it's not an exact translation, so it's up to Rugk.

---

It's nozt really up to me, I don't speak Frensh… :upside_down_face: 

So better you decide. You can click on "Resolve conversation" when this is finished, BTW, to show the issue has been tackled.

---

Il se dédouane du choix à faire, toi comme moi on préfère parler de « personnes » plutôt que de devoir jongler avec « utilisateur » et personne ne s’y est opposé. On part là dessus !

---

Let's go for "personnes", then :+1: